text/0057-update-mixin-uniqueness-rules-for-scis.md

```diff
@@ -0,0 +1,146 @@
+- Feature Name: update-mixin-uniqueness-rules-for-scis
+- Start Date: 2022-11-03
+- MCIP PR: [mobilecoinfoundation/mcips#0057](https://github.com/mobilecoinfoundation/mcips/pull/0057)
+- Tracking Issue: [mobilecoinfoundation/mobilecoin#2813](https://github.com/mobilecoinfoundation/mobilecoin/issues/2813)
+
+# Summary
+[summary]: #summary
+
+Update the transaction validation rules around  "DuplicateRingElements" error,
+relaxing them when MCIP #31 Signed Contingent Inputs are present in the transaction.
+
+# Motivation
+[motivation]: #motivation
+
+Currently, the consensus enclave enforces the following rule: a ring element cannot appear in two different inputs in the transaction.
+
+If this rule is violated, the transaction fails with a "DuplicateRingElements" error.
+
+The rationale for this rule is
+* It is generally possible for clients to sample the mixin ring elements without replacement,
+  removing all the real inputs, and so ensure that duplicate ring elements do not occur.
+* Doing so may make it more difficult for an adversary to statistically analyze the rings.
+
+This only justifies why this rule should be a **best practice for clients**, but not why
+it should be enforced at the level of the consensus network. In fact, this rule is not needed
+to protect the integrity of the cryptocurrency -- it doesn't allow double spends or imbalanced
+transactions to remove this rule. It simply helps to prevent clients from using bad ring sampling
+code in production.
```

Comment:
```suggestion
to protect the integrity of the cryptocurrency -- removing it doesn't allow double spends
or imbalanced transactions. It simply helps to prevent clients from using bad ring sampling
code in production.
```